openapi/publicapi_spec.json

```diff
@@ -220,14 +220,14 @@
             "type" : "string"
           }
         }, {
-          "description" : "From date of payments to be searched (this date is inclusive). Example=2015-08-13T12:35:00Z",
+          "description" : "From date of payments to be searched (this date is inclusive). Example=2015-08-13T12:35:00.000Z",
```

Comment:
This is a bit more interesting: while we also include milliseconds for outputs in our API responses, this is an input we will allow something like `2015-08-13T12:35:00Z` there. Does that cause you problems?